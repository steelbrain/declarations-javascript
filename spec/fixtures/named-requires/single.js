const { moduleName: anotherName } = require('file-moduleName')
anotherName()
