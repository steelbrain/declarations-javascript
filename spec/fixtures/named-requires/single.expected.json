[
  {
    "name": "anotherName",
    "position": {
      "start": {
        "line": 1,
        "column": 20
      },
      "end": {
        "line": 1,
        "column": 31
      }
    },
    "source": {
      "name": "moduleName",
      "filePath": "file-moduleName",
      "position": {
        "start": {
          "line": 1,
          "column": 44
        },
        "end": {
          "line": 1,
          "column": 61
        }
      }
    }
  },
  {
    "name": "yeahZ",
    "position": {
      "start": {
        "line": 2,
        "column": 8
      },
      "end": {
        "line": 2,
        "column": 13
      }
    },
    "source": {
      "name": "yeahZ",
      "filePath": "z",
      "position": {
        "start": {
          "line": 2,
          "column": 26
        },
        "end": {
          "line": 2,
          "column": 29
        }
      }
    }
  },
  {
    "name": "anotherName",
    "position": {
      "start": {
        "line": 3,
        "column": 0
      },
      "end": {
        "line": 3,
        "column": 11
      }
    },
    "source": {
      "name": null,
      "filePath": null,
      "position": {
        "start": {
          "line": 1,
          "column": 20
        },
        "end": {
          "line": 1,
          "column": 31
        }
      }
    }
  }
]